targets/bcm27xx-bcm2711

```diff
@@ -0,0 +1,7 @@
+include 'bcm27xx.inc'
+
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
+		'raspberry-pi-4-model-b-rev-1.1',
```

Comment:
We have a bunch more devices currently, in particular rev-1.4

Reply:
Here are the variants we have:
https://github.com/freifunk-gluon/gluon/pull/3073/files#diff-354a477dd339a057cbdd0607732186aa6c57d53f65c954dc41da432c580ec225R5-R9

We do not run Gluon v2019 anymore though, so I don't think all of them are required for us at least

Reply:
Image names for all RPi targets are currently broken https://github.com/freifunk-gluon/gluon/issues/3099 - we can drop all these aliases when that has been fixed.